src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/SchedulerElinksJobIntegrationTest.java

```diff
@@ -84,10 +94,8 @@ void test_load_elinks_job_status_failure() {
         elinksApiJobScheduler.loadElinksJob();
 
         List<DataloadSchedulerJob> audits = dataloadSchedulerJobRepository.findAll();
-        DataloadSchedulerJob jobDetails = dataloadSchedulerJobRepository.findAll().get(0);
 
-        assertThat(jobDetails).isNotNull();
-        assertThat(jobDetails.getPublishingStatus()).isNotNull();
```

Comment:
same as above

Reply:
It will be covered as a part of other userstory